lms/static/styles/moodle_pages/moodle_pages.scss

```diff
@@ -0,0 +1,226 @@
+body {
```

Comment:
The `canvas_pages.scss` module for Canvas has some comments at the top has some notes about where the styles came from and upstream links that are useful if we need to update it in future. I think it would be useful to have  something similar here. In the case of fonts, it looks like Moodle is relying on system fonts, so we don't need any imports for those.

Reply:
I have added a comment to the top, referencing the main files which should be checked in case new updates are needed.